bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -80,6 +80,10 @@ struct ReplicaConfig {
   // StateTransfer
   uint32_t maxNumOfReservedPages = 2048;
   uint32_t sizeOfReservedPage = 4096;
+
+  // If set to true, this replica will periodically log debug statistics such as
+  // throughput and number of messages sent.
+  bool debugStatisticsEnabled = false;
```

Comment:
I am aware that I am changing the default behavior here. It was my intention to make the debug statistics disabled by default, as I perceived them as something we used in development and debugging, but not something that we would actually want in production (though perhaps I am making assumptions I shouldn't be based on the fact these are called the "debug" statistics). It was my thinking that we should probably make intended production behavior the default rather than debug behavior as the default. I cannot tell whether your comment is just asking me to confirm that disabling this by default was my intention or whether you intend to imply the maintainers of Concord-BFT need debug statistics to be enabled by default.

If you want me to leave this enabled by default, I can work with that. I have a production use case from a consumer of this library where we probably want debug statistics disabled, but I can make that consumer elect the non-default value here if you want to make enabled default.

Reply:
I wasn't sure if you did it on purpose. Since you did this on purpose, I personally don't use it, I  don't believe it's heavily used elsewhere, and we have a customer who wants it off, I think we can leave it off. It's easy enough to configure for tests.

Reply:
I am going to err on the side of leaving the statistics disabled by default if no one more explicitly objects to this.